Content.Shared/Weapons/Ranged/Components/GunComponent.cs

```diff
@@ -14,16 +14,16 @@ public partial class GunComponent : Component
 {
     #region Sound
 
-    [ViewVariables(VVAccess.ReadWrite), DataField("soundGunshot")]
```

Comment:
since you changing these anyway, datafields are automatically rw now so you can just do [DataField]

Reply:
I knew about the string, but not the automatic rw